vector/src/main/res/layout/view_room_detail_toolbar.xml

```diff
@@ -82,8 +82,8 @@
         android:id="@+id/roomToolbarSubtitleView"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_marginStart="7dp"
-        android:layout_marginEnd="8dp"
+        android:layout_marginStart="12dp"
+        android:layout_marginEnd="12dp"
```

Comment:
I think you can just delete this View (with id `roomToolbarSubtitleView`), it's always hidden now, so it's dead code.